core/src/main/java/tc/oc/pgm/modules/ModifyBowProjectileModule.java

```diff
@@ -59,11 +69,17 @@ public ModifyBowProjectileModule parse(MapFactory factory, Logger logger, Docume
           potionEffects.add(XMLUtils.parsePotionEffect(potionElement));
           changed = true;
         }
+
+        Element noPickupElement = parent.getChild("pickup");
+        if (noPickupElement != null) {
+          pickupFilter = filters.parseFilterProperty(noPickupElement, "filter");
+          changed = true;
+        }
```

Comment:
This changes doesn't actually enforce there being a unique child, and it feels like the easier way to handle this would be by making the filter a property of modifybowprojectile, it would make the syntax be:
```xml
<modifybowprojectile pickup-filter="x"/>
```
or
```xml
<modifybowprojectile>
  <pickup-filter>...</pickup-filter>
</modifybowprojectile>
```
which goes in-line with how it's generally done in pgm, since you just want one child why not inline it in the parent.

```suggestion
        Filter pickupFilter = filters.parseFilterProperty(parent, "pickup-filter", StaticFilter.ALLOW);
        changed |= pickupFilter != StaticFilter.ALLOW;
```

This would enforce single-child properly and simplify the syntax